docs/DATA_TYPES.md

```diff
@@ -24,7 +24,7 @@ Data type constants have some naming (and value) rules.
 STREAM content is generally pulled from an Ingestion Engine and passed to the countertop.
 
 * `STREAM.ALL` - Any type of stream data.
-* `STREAM.DATA` - Buffers representing a portion of data stream.
+* `STREAM.CONTAINER` - Buffers representing a portion of a mpegts data stream.
```

Comment:
I would really love if this read `an MPEG-TS data stream` because… I'm like that. 😬